<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tar_backup</class>
 <widget class="QMainWindow" name="tar_backup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tar Backup Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="backuptab">
       <attribute name="title">
        <string>Backup</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <italic>false</italic>
           </font>
          </property>
          <property name="title">
           <string>Backup profiles</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="3" column="2">
            <widget class="QPushButton" name="btn_removeProfile">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="5" column="2">
            <widget class="QPushButton" name="btn_run">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>41</height>
              </size>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Run</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="btn_addProfile">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
               <italic>false</italic>
              </font>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="btn_modifyProfile">
             <property name="text">
              <string>Modify</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="6">
            <widget class="QListWidget" name="list_backupProfiles">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionRectVisible">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="restoreTab">
       <attribute name="title">
        <string>Restore</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QPushButton" name="btn_selectFileRestore">
          <property name="minimumSize">
           <size>
            <width>191</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>170</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Select file to restore...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="t_restoreFile">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Restore backup</string>
           </attribute>
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="1" column="0">
             <widget class="QPushButton" name="btn_setDestRestore">
              <property name="minimumSize">
               <size>
                <width>191</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>Set destination...</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="cb_decryptOnly">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Decrypt only</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QLineEdit" name="t_destRestore">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="btn_runRestore">
              <property name="minimumSize">
               <size>
                <width>191</width>
                <height>41</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>191</width>
                <height>41</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Restore</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>List members</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="1" column="0" colspan="5">
             <widget class="QPlainTextEdit" name="tarMembersT">
              <property name="font">
               <font>
                <family>DejaVu Sans Mono</family>
               </font>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="btn_listMembers">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>28</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>List members</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="btn_saveMembersToFile">
              <property name="minimumSize">
               <size>
                <width>111</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>111</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Save to file</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="cb_listExtendedInfo">
              <property name="text">
               <string>Extended info</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="outputTab">
       <attribute name="title">
        <string>Output</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="4" column="0">
         <widget class="QPushButton" name="btn_abort">
          <property name="text">
           <string>Abort</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="label_status">
          <property name="font">
           <font>
            <family>DejaVu Sans</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Status</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="label_process">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="btn_saveOutput">
          <property name="text">
           <string>Save to file</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QPlainTextEdit" name="outputT">
          <property name="font">
           <font>
            <family>DejaVu Sans</family>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string>console output</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
